lib/controller.js

```diff
@@ -239,17 +239,28 @@ Tessel.get = function(opts) {
         }
       }
 
+
+      function finishSearchEarly(tessel) {
+        // Remove this listener because we don't need to search for the Tessel
+        seeker.removeListener('end', searchComplete);
+        // Stop searching
+        seeker.stop();
+        // Send this Tessel back to the caller
+        logAndFinish(tessel);
```

Comment:
> does it need that even though seeker is within that function's scope already?

I actually hadn't looked at the entire file to see the context around the declaration and call site. Seeing the entire thing, it looks fine. Sorry for the noise there.